### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
##Summary
add Responsive design

##Screenshot
captura de pantalla 2018-09-13 a la s 16 16 10

##Trello card
https://trello.com/c/pd6LZe66/15-dise%C3%B1o-responsive

##Video
[video4.zip](https://github.com/wolox-training/gs-react-native/files/2380536/video4.zip)

### CODE REVIEW

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Check the header, I guess the footer too. Try to send a gif or video where we can see the entire screen scrolling.

File: scss/navbar.scss [hunk 1]
```diff
@@ -22,3 +22,13 @@
   margin-right: 50px;
   font-size: 30px;
 }
+
+@media (max-width: 1024px) {
+  .title-navbar {
+    color: $color-white;
+  }
+
+  .image-navbar{
```
Comment (FrankIglesias) [inline scss/navbar.scss]: add a space before the curly brace

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/footer.scss [hunk 2]
```diff
@@ -20,3 +20,14 @@
...
 }
+
+@media (max-width: 1024px) {
+  .text-copyright {
+    align-self: center;
+  }
+
+  .img-footer {
+    max-width: 300px; 
+    max-height: 300px;
```
Comment (mmamaniwolox) [inline scss/footer.scss]: You only need one of those

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Technologies are not being centered in small screens

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Check footer width

File: scss/footer.scss [hunk 3]
```diff
@@ -5,7 +5,8 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  margin-bottom: 10px;
+  top: 0px;
+  left: 0px;
```
Comment (mmamaniwolox) [inline scss/footer.scss]: Use position: absolute if you will use top and left

File: scss/navbar.scss [hunk 4]
```diff
@@ -22,3 +22,13 @@
...
   font-size: 30px;
 }
+
+@media (max-width: 1024px) {
+  .title-navbar {
+    color: $color-white;
+  }
+
+  .image-navbar {
+    height: auto;
```
Comment (mmamaniwolox) [inline scss/navbar.scss]: I dont recommend this, images should always have a fixed height

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/tech.scss [hunk 5]
```diff
@@ -53,3 +52,24 @@
...
+@media (max-width: 1024px) {
+  .container-technologies {
+    margin-left: 0;
+    display: grid;
+    grid-template-columns: 1fr;
+    justify-content: center;
+  }
+
+  .techno-content {
+    border-right-color: $color-white;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: This is to hide boder? use `border-right: 0`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**